content/pt-br/community/static/cncf-code-of-conduct.md

```diff
@@ -13,32 +14,62 @@ todos, independentemente do nível de experiência, gênero, identidade e expres
+Conduta podem ser removidos permanentemente da equipe do projeto.
 
-Este código de conduta aplica-se tanto nos espaços do projeto como nos espaços públicos
-quando um indivíduo está representando o projeto ou sua comunidade.
 
-Casos de comportamento abusivo, de assédio ou inaceitável no Kubernetes podem ser relatados entrando em contato com o [Comitê de Código de Conduta da Kubernetes] (https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. Para outros projetos, entre em contato com um mantenedor do projeto CNCF ou com nosso mediador, Mishi Choudhary <mishi@linux.com>.
+## Reportando 
 
-Este Código de Conduta é adaptado do Pacto do Colaborador
-(https://contributor-covenant.org), versão 1.2.0, disponível em
-https://contributor-covenant.org/version/1/2/0/
+Para incidentes ocorridos na comunidade Kubernetes, entre em contato com o [Comitê de Código de Conduta da Kubernetes](https://git.k8s.io/community/committee-code-of-conduct) via <conduct@kubernetes.io>. Você pode esperar uma resposta dentro de três dias úteis.
+
+Para outros projetos, ou para incidentes independentes do projeto ou que afetem vários projetos do CNCF, entre em contato com o [Comitê de Código de Conduta CNCF](https://www.cncf.io/conduct/committee/) via conduct@cncf.io. Como alternativa, você pode entrar em contato com qualquer um dos membros individuais do [Comitê de Código de Conduta CNCF](https://www.cncf.io/conduct/committee/) para enviar seu relato. Para obter instruções mais detalhadas sobre como enviar um relato, incluindo como enviar um relato anonimamente, consulte nosso [Procedimentos de Resolução de Incidentes](https://www.cncf.io/conduct/procedures/). Você pode esperar uma resposta dentro de três dias úteis.
+
+## Aplicação 
 
-### Código de Conduta para a Comunidade CNCF
+O [Comitê do Código de Conduta](https://github.com/kubernetes/community/tree/master/committee-code-of-conduct) do projeto Kubernetes aplica questões de código de conduta para o projeto Kubernetes.
 
-Os eventos CNCF são regidos pelo Linux Foundation [Código de Conduta] (https://events.linuxfoundation.org/code-of-conduct/) disponível na página do evento. Isso foi projetado para ser compatível com a política acima e também inclui mais detalhes sobre como responder a incidentes.
\ No newline at end of file
+Para todos os projetos que não têm seu próprio Comitê do Código de Conduta ou outros respondentes de incidentes do Código de Conduta, e para incidentes que são independentes do projeto ou afetam vários projetos, o [Comitê de Código de Conduta CNCF](https://www.cncf.io/conduct/committee/) impõe questões de código de conduta. Para obter mais informações, consulte nossa [Política de Jurisdição e Escalação](https://www.cncf.io/conduct/jurisdiction/)
+
+Ambos os órgãos tentam resolver as ocorrências sem punições, mas podem retirar pessoas do projeto ou das comunidades do CNCF a seu critério.
+
+## Agradecimentos
+
+Este Código de Conduta é adaptado do Pacto do Colaborador
```

Comment:
It's better to keep Contributor Covenant non-translated here (see https://www.contributor-covenant.org/pt-br/version/2/0/code_of_conduct/).

Reply:
Thank you again for your feedback, @stormqueen1990! I fixed the issues with `content/pt-br/community/static/cncf-code-of-conduct.md` at [328269d](https://github.com/kubernetes/website/pull/38407/commits/328269d7a6d456704c77277de780cf744bd3c60c)